src/Razor/src/rzls/PublishAllRids.targets

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <!-- We are defining our own Pack target, so opt out of the SDK version. -->
+  <PropertyGroup>
+    <ImportNuGetBuildTasksPackTargetsFromSdk>false</ImportNuGetBuildTasksPackTargetsFromSdk>
+    <_RazorPublishReadyToRun>false</_RazorPublishReadyToRun>
+    <_RazorPublishReadyToRun Condition="'$(Configuration)' == 'Release'">true</_RazorPublishReadyToRun>
+  </PropertyGroup>
+
+  <Target Name="Pack">
+    <!--
+        We have to run restore first with the PublishReadyToRun flag set to true to ensure that the correct crossgen packages get restored.
+        See https://github.com/dotnet/sdk/issues/20701
+
+        We also pass the RestoreUseStaticGraphEvaluation=false flag to workaround a long path issue when calling the restore target.
+        See https://github.com/NuGet/Home/issues/11968
+    -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="Restore" Properties="PublishReadyToRun=$(_RazorPublishReadyToRun);RestoreUseStaticGraphEvaluation=false" />
+
+    <ItemGroup>
+      <!-- Transform RuntimeIdentifiers property to item -->
+      <RuntimeIdentifierForPack Include="$(RuntimeIdentifiers)" />
+      <RuntimeIdentifierForPack Include="neutral" />
+
+      <!-- Transform RuntimeIdentifierForPublish items to project items to pass to MSBuild task -->
+      <ProjectToPublish Include="@(RuntimeIdentifierForPack->'$(MSBuildProjectFullPath)')">
+          <AdditionalProperties>PackRuntimeIdentifier=%(RuntimeIdentifierForPack.Identity)</AdditionalProperties>
```

Comment:
Tiniest of nits! It looks to me like this got an extra indent.

Reply:
🙀

Will fix :)

Reply:
I swear that I did not review your code with a ruler.

Reply:
That's okay, I got one for you (yes I installed powertoys for this joke)

![image](https://github.com/dotnet/razor/assets/475144/ac844731-0734-4a39-af26-61450a2347d1)